x-pack/test/ingest_manager_api_integration/apis/list.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../api_integration/ftr_provider_context';
+import { warnAndSkipTest } from '../helpers';
+
+export default function ({ getService }: FtrProviderContext) {
+  const log = getService('log');
+  const supertest = getService('supertest');
+  const dockerServers = getService('dockerServers');
+
+  const server = dockerServers.get('registry');
+  // use function () {} and not () => {} here
```

Comment:
Thank you so much for this note. I've "discovered" it a few times in my career. Thanks from Future Me for the found time and mental health.